examples/01_icetray/04_i3_module_in_native_icetray_example.py

```diff
@@ -71,7 +72,10 @@ def main() -> None:
     # Constants
     features = FEATURES.UPGRADE
     input_folders = [f"{TEST_DATA_DIR}/i3/upgrade_genie_step4_140028_000998"]
-    mock_model_path = f"{TEST_DATA_DIR}/models/mock_energy_model.pth"
+    base_path = f"{PRETRAINED_MODEL_DIR}/icecube/upgrade/QUESO"
+    model_name = "total_neutrino_energy"
+    model_config = f"{base_path}/{model_name}/{model_name}_config.yml"
+    state_dict = f"{base_path}/{model_name}/{model_name}_state_dict.yml"
```

Comment:
Same as the previous comment about state_dicts.

Reply:
Thanks. The dangers of copy paste...